### PR TITLE
V6/update how it works layout

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Remove mew-tabs
   - Update how it works layout

Screenshot_2021-03-22 MyEtherWallet com Your Key to Ethereum(1)

Screenshot_2021-03-22 MyEtherWallet com Your Key to Ethereum(2)

### CODE REVIEW

File: src/views/layouts-default/TheHowItWorksLayout.vue [hunk 1]
```diff
@@ -1,23 +1,20 @@
 <template>
   <div class="mew-component--how-it-works">
     <the-layout-header title="How it works" />
-    <mew-tabs :items="mewTabs" has-underline>
```
Comment (olgakup) [inline src/views/layouts-default/TheHowItWorksLayout.vue:4]: why did you remove tab menu?

Comment (hsurf22) [reply src/views/layouts-default/TheHowItWorksLayout.vue:4]: Alex talked about it few weeks ago about this on Tuesday meeting and we all agreed to remove tab and show all contents on one page.
